framework-test/src/main/java/org/checkerframework/framework/test/TestConfigurationBuilder.java

```diff
@@ -73,6 +78,13 @@ public static TestConfigurationBuilder getDefaultConfigurationBuilder(
                 .addOption("-implicit:class")
                 .addOption("-classpath", classPath);
 
+        // -Anomsgtext is needed to ensure expected errors can be matched, which is the
+        // right thing for most test cases.
+        configBuilder.addOption("-Anomsgtext");
```

Comment:
Should we mention `removeConflicts` somewhere here, to make a reader aware of the conflict checks?

Reply:
Yes, comment added.